ci/pipelines/deploy.yml

```diff
@@ -0,0 +1,78 @@
+resources:
+  - name: govwifi-admin
+    type: git
+    source:
+      uri: "https://github.com/alphagov/govwifi-admin.git"
+      branch: "concourse"
+
+  - name: ecr
+    # This needs to be docker-image, until registry-image supports ECR
```

Comment:
Not entirely sure what this comments means.
ECR isn't a supported registry?

Reply:
https://github.com/concourse/registry-image-resource/pull/27

feature is being worked on.
Concourse recommends `registry-image` over `docker-image`

Reply:
Hoping to get it merged soon. In the meantime you could use:
```
- name: registry-image-resource
  type: docker-image
  source:
    repository: dwpdigital/registry-image-resource
```

Reply:
Thanks for that @dan-hill2802. 
For now, we'll stick with using `docker-image` until the functionality is mainlined.
There's no pressing need to use `registry-image`